rust/lance-index/Cargo.toml

```diff
@@ -49,6 +49,8 @@ snafu.workspace = true
 tokio.workspace = true
 tracing.workspace = true
 tempfile.workspace = true
+kdam = "0"
```

Comment:
kdam was useful for watching builds, but it should probably be removed so we don't spam stdout.